README.md

```diff
@@ -1,5 +1,7 @@
 # community-topics
 
+![](https://img.shields.io/matrix/community:ansible.com.svg?server_fqdn=ansible-accounts.ems.host&label=Discuss%20at%20%23community:ansible.com&logo=matrix)
```

Comment:
There should also be some usable alt text IMO:
```suggestion
![Discuss at #community:ansible.com on Matrix](https://img.shields.io/matrix/community:ansible.com.svg?server_fqdn=ansible-accounts.ems.host&label=Discuss%20at%20%23community:ansible.com&logo=matrix)
```

Reply:
yep, on Matrix should be added. We've already done it too in c.mysql and c.postgresql READMEs